### PR TITLE
ADD: Time tolerance to GateMapper

### PR DESCRIPTION
**Pull request by rcjackson**
Added a time tolerance to the GateMapper function. Now you can exclude points that are out of synchronization when doing gate to gate comparisons.

### CODE REVIEW

**Review by mgrover1 — CHANGES_REQUESTED** [review]
Small name change

**Comment by mgrover1** [comment]
@rcjackson just a couple of suggestions in terms of naming... I think this will make it clearer for people using this API

File: pyart/map/gate_mapper.py [hunk 1]
```diff
@@ -50,32 +50,43 @@ class GateMapper(object):
         The gatefilter to apply to the source radar data before mapping
     tol: float
         The difference in meters between the source and destination gate allowed for an adequate match.
+    time_tol: float
```
[the remark this answers is not in the document]
Comment (rcjackson) [reply pyart/map/gate_mapper.py]: I've put in a new commit changing the names of these two variables to be more explicit throughout.

[the remark this answers is not in the document]
Comment (mgrover1) [reply pyart/map/gate_mapper.py]: Awesome - the example will need an update too... which is why the doc build is failing

**Comment by mgrover1** [comment]
@rcjackson I added the example doc update here since the api changed slightly

**Review by mgrover1 — APPROVED** [review]
Alrighty! This looks good to go 👍

**Pull request merged**